### PR TITLE
Some small fixes for editor-revision-2021-03-23

### PR DESCRIPTION
**Pull request by tlimmer**
Contains a few small fixes for editor-revision-2021-03-23.

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -1647,7 +1651,7 @@ Valid values are:
     not_contacted
     open
 
-Summary of involvement (`summary`) of value type `string` with 1 or more characters contains additional context regarding what is going on.
+Summary of involvement (`summary`) of value type `string` with 1 or more characters contains optional additional context regarding what is going on.
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1654]: That `summary` is optional was already mentioned in line 1621. We should not double that.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 2]
```diff
@@ -1779,7 +1783,7 @@ Restart required by remediation (`restart_required`) of value type `object` with
         }
       },
 
-Additional restart information (`details`) of value type `string` with 1 or more characters provides additional information for the restart. This can include details on procedures, scope or impact.
+Optional restart information (`details`) of value type `string` with 1 or more characters provides additional information for the restart. This can include details on procedures, scope or impact.
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1786]: The JSON schema states the title as "Additional restart information". This is unrelated to the fact that `details` is an optional property (see line 1712).

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 3]
```diff
@@ -301,6 +301,8 @@ _Data elements and interchange formats — Information interchange — Represent
 ###### [ISO29147]
 _Information technology — Security techniques — Vulnerability disclosure_, International Standard, ISO 29147:2014(E), February 15, 2014,
 https://www.iso.org/standard/45170.html.
+###### [PURL]
+_Package URL (PURL)_, Github Project, https://github.com/package-url/purl-spec
```
Comment (mprpic) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:305]:
```suggestion
_Package URL (PURL)_, GitHub Project, https://github.com/package-url/purl-spec
```

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 4]
```diff
@@ -640,7 +645,7 @@ Product ID (`product_id`) holds a value of type Product ID (`product_id_t`).
 #### 3.1.3.3 Full Product Name Type - Product Identification Helper
 
 Helper to identify the product (`product_identification_helper`) of value type `object` provides in its properties at least one method which aids in identifying the product in an asset database.
-Of the given five properties `cpe`, `hashes`, `purl`, `serial_numbers`, and `x_generic_uris` one is mandatory.
+Of the given five properties `cpe`, `hashes`, `purl`, `serial_numbers`, and `x_generic_uris`, at least one is MUST be given.
```
Comment (mprpic) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:648]:
```suggestion
Of the given five properties `cpe`, `hashes`, `purl`, `serial_numbers`, and `x_generic_uris`, at least one MUST be provided.
```

**Review by mprpic — APPROVED** [review]
Just a couple of tiny improvements. LGTM!

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 5]
```diff
@@ -728,7 +733,7 @@ Examples:
 
 ##### 3.1.3.3.3 Full Product Name Type - Product Identification Helper - PURL
 
-The package URL (PURL) representation (`purl`) is a `string` of 4 or more characters with `pattern` (regular expression):
+The package URL (PURL) representation (`purl`) [PURL] is a `string` of 4 or more characters with `pattern` (regular expression):
```
[the remark this answers is not in the document]
Comment (tlimmer) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:736]: Added the PURL reference in line 740, added CPE reference in line 676.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 6]
```diff
@@ -1019,15 +1024,15 @@ In addition, the `document` object may provide the 7 optional properties Acknowl
...
 
     "acknowledgments": {
       // ...
     },
 
 #### 3.2.1.2 Document Property - Aggregate Severity
 
-Aggregate severity (`aggregate_severity`) of value type `object` with the mandatory property `text` and the optional property `namespace` is a vehicle that is provided by the document producer to convey the urgency and criticality with which the one or more vulnerabilities reported should be addressed. It is a document-level metric and applied to the document as a whole — not any specific vulnerability. The range of values in this field is defined according to the document producer's policies and procedures.
+Aggregate severity (`aggregate_severity`) of value type `object` with the mandatory property `text` and the optional property `namespace` is a vehicle that is provided by the document producer to convey the urgency and criticality with which the reported issues should be addressed. It is a document-level metric and applied to the document as a whole — not any specific vulnerability. The range of values in this field is defined according to the document producer's policies and procedures.
```
[the remark this answers is not in the document]
Comment (tlimmer) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1035]: Schema updated.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 7]
```diff
@@ -718,7 +723,7 @@ The Value of the cryptographic hash representation (`value`) of value type `stri
 
     ^[0-9a-fA-F]{64,}$
 
-The Value of the cryptographic hash attribute contains the cryptographic hash value.
+The Value of the cryptographic hash attribute contains the cryptographic hash value in hexadecimal representation.
```
[the remark this answers is not in the document]
Comment (tlimmer) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:726]: Changed in schema

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1035]: @mprpic, @sthagen, @santosomar: Please provide feedback.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 8]
```diff
@@ -668,7 +673,7 @@ Common Platform Enumeration representation (`cpe`) of value type `string` of 5 o
 
     ^(cpe:2\\.3:[aho\\*\\-](:(((\\?*|\\*?)([a-zA-Z0-9\\-\\._]|(\\\\[\\\\\\*\\?!\"#\\$%&'\\(\\)\\+,/:;<=>@\\[\\]\\^`\\{\\|\\}~]))+(\\?*|\\*?))|[\\*\\-])){5}(:(([a-zA-Z]{2,3}(-([a-zA-Z]{2}|[0-9]{3}))?)|[\\*\\-]))(:(((\\?*|\\*?)([a-zA-Z0-9\\-\\._]|(\\\\[\\\\\\*\\?!\"#\\$%&'\\(\\)\\+,/:;<=>@\\[\\]\\^`\\{\\|\\}~]))+(\\?*|\\*?))|[\\*\\-])){4})|([c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\\._\\-~%]*){0,6})$
 
-The Common Platform Enumeration (CPE) attribute refers to a method for naming platforms external to this specification.
+The Common Platform Enumeration (CPE) [CPE23-A] attribute refers to a method for naming platforms external to this specification.
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:676]:
```suggestion
The Common Platform Enumeration (CPE) attribute refers to a method for naming platforms external to this specification. See [CPE23-A] for details.
```

Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:676]: @sthagen: Please provide feedback.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 9]
```diff
@@ -732,13 +737,13 @@ The package URL (PURL) representation (`purl`) is a `string` of 4 or more charac
 
     ^pkg:
 
-This package URL (PURL) attribute refers to a method for reliably identifying and locating software packages external to this specification.
+This package URL (PURL) [PURL] attribute refers to a method for reliably identifying and locating software packages external to this specification.
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:740]:
```suggestion
This package URL (PURL) attribute refers to a method for reliably identifying and locating software packages external to this specification. See [PURL] for details.
```

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1035]: Three times then :wink:. @tlimmer Please "keep" `one or more vulnerabilities` because then we can simply merge, and I can continue to work on the document/schema - thanks.

[the remark this answers is not in the document]
Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1035]: Or maybe this way: When tomorrow afternoon European time this PR is still using the "issues" term in here, I will simply merge to be able to continue collecting all other comments into the main revision. Thanks.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Same here, I offer to merge and then adapt in my next revision.

**Review by sthagen — APPROVED** [review]
I gave feedback in several places and kindly suggest to merge as is so I can continue to collect the comments received not yet included in this PR.

**Review by sthagen — APPROVED** [review]
I think resolved or scoped and pending - will merge to continue for now. Thanks for providing the changes. Now I will have to ensure no comment got lost.

**Review by sthagen — APPROVED** [review]
LGTM

**Comment by tschmidtb51** [comment]
Will get the changes in soon.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 10]
```diff
@@ -1019,15 +1024,15 @@ In addition, the `document` object may provide the 7 optional properties Acknowl
 
 #### 3.2.1.1 Document Property - Acknowledgments
 
-List of acknowledgments (`acknowledgments`) of value type `array` with 1 or more items of type Acknowledgment contains a list of acknowledgment elements.
+List of acknowledgments (`acknowledgments_t`) of value type `array` with 1 or more items of type Acknowledgment contains a list of acknowledgment elements.
```
[the remark this answers is not in the document]
Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Will ensure in next revision

**Comment by sthagen** [comment]
> Will get the changes in soon.

Shoot. We crossed each other by seconds. Please feel free to provide your pending changes @tschmidtb51

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Is reverted with #206.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1035]: Is reverted with #206.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 9 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:740]: Was copied to #205 to have the discussion in a single place.
